pkg/pods/pod.go

```diff
@@ -229,11 +229,12 @@ func (pod *Pod) buildRunitServices(launchables []launch.Launchable, restartPolic
 				return util.Errorf("Duplicate executable %q for launchable %q", executable.Service.Name, launchable.ID())
 			}
 			sbTemplate[executable.Service.Name] = runit.ServiceTemplate{
+				Log: newManifest.GetLogExecutable(),
```

Comment:
After a discussion offline: we decided that the gradual rollout will be managed by the Square linkage to this code. p2-log-bridge will be the logger for all applications henceforth.

This means that all regular STDOUT logs will be unnecessarily buffered  (in a potentially leaky way) in the open source impl.